README.md

```diff
@@ -132,7 +132,7 @@ If it's easier you can also run a relay with Docker:
 
 
 ```
-$ docker run -d -p 9009:9009 -p 9010:9010 -p 9011:9011 -p 9012:9012 -p 9013:9013 schollz/croc
+$ docker run -d -p 9009:9009 -p 9010:9010 -p 9011:9011 -p 9012:9012 -p 9013:9013 -e CROC_PASS YOURPASSWORD schollz/croc

```

Comment:
I think this is missing an = between CROC_PASS and YOURPASSWORD

Reply:
Fixed.